docs/examples/make_pymvpa_collection.sh

```diff
@@ -0,0 +1,80 @@
+#!/bin/sh
+# A little script to prepare PyMVPA demo collection based on data used in
+# tutorials etc
+
+set -eu
+
+echo "I: adding haxby2001"
+# Haxby 2001 -- the main demo dataset
+datalad install-handle http://data.pymvpa.org/datasets/haxby2001/.git
+
+# getting its content, since publishing can't clone the repo, we need to push
+# content to the published handle and therefore we would loose the connection
+# to content, that is not addurl'ed but just available via the original annex.
+# Note: May be therefore use "git annex get . --from=origin" to not unnecessarily
+# get things, that are reachable from the published handle (and any clone of
+# it)? Need to add that option to datalad get.
```

Comment:
Several questions/issues:
1. Why a special remote? Because of the autoenabling? With current publishing methods (ssh or local FS) we could also just add the origin as a usual git remote to the published handle.
2. Is there a special remote type, that can be used with a plain annex?
3. Either way, this would be kind of a special setup for publishing. Regarding a general approach - how to determine what (special) remotes to add, when publishing a handle? So, use that approach just for generating a published pymvpa collection? In that case we can simply go for the git remote to the original repo.
4. It's not implemented in the "add-handle" command yet, but in underlying repo-class: We can add a handle to the collection, that is not locally available. So, if there's nothing to be changed in the handle itself, we don't even need to install and then publish it with the collection in first place. At least, if the origin is publicly available.
5. After all, depends on what's the main purpose of that script: Demonstration for users to look at or just getting that collection generated and published?